docs/examples/ts/webpack.config.js

```diff
@@ -0,0 +1,43 @@
+const path = require('path');
+const tsFiles = [
```

Comment:
Why do we need that?

Reply:
To make sure that example.ts will be the last one in the list of files. Files path changed.